yarn.lock

```diff
@@ -7431,9 +7430,9 @@ uglify-to-browserify@~1.0.0:
   integrity sha1-bgkk1r2mta/jSeOabWMoUKD4grc=
 
 uikit@^3.0.0-rc.9:
```

Comment:
we still have uikint in here?

Reply:
demo, and servstat app still depend on uikit asap we throw them out or rebuild them this is gone